src/GeoJSONTables.jl

```diff
@@ -35,7 +35,7 @@ function readtable(source)
     if get(fc, :type, nothing) == "FeatureCollection" && jsonfeatures isa JSON3.Array
         for f in jsonfeatures 
             prop = f.properties
-            if !=(get(f, :properties, nothing), nothing)
+            if get(f, :properties, nothing) != nothing
```

Comment:
For `nothing` it can be faster to do `=== / !==` versus `== / !=`. You can also use `isnothing`. Note in the lines below there are still a few `!=(` cases as well.